ci/pipelines/test-ecr-push.yml

```diff
@@ -0,0 +1,186 @@
+definitions:
+  # ECR Test Registry Definition
+  # Use <<: *ecr-test-registry and update the name and source/repository as
+  # necessary.
+  - &ecr-test-registry
```

Comment:
I like this! Although I think we might be doing something different with ready-staging